MTMImporter/Importer.cs

```diff
@@ -318,21 +320,32 @@ public static void Main(string[] args)
         {
             try
             {
-                ValidateArguments(args);
-                var uri = args[0];
-                var username = args[1];
-                var password = args[2];
-                var domain = args[3];
-                var teamProject = args[4];
-                Parser parser = new Parser();
-                var resultsSession = parser.Parse(args[5]);
-                var testRun = ConvertResultToTestRun(resultsSession);
-                var importer = new Importer(uri, username, password, domain);
-                importer.Import(teamProject, testRun);
+                bool isEncrypt = args.Length == 2 && args[0] == "--encrypt";

```

Comment:
You should probably use "-encodePass" to be consistent with the other tools. Also, why not use the options parser for this too? It doesn't really make sense to do it both ways. If this option is passed then it should just generate the password and ignore any other options, or at least behave consistently with the analyzers.